Flask_lesson#3_HW_10/staff/main_staff.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint
+from flask_restful import Api
+
+from staff.staff_resourse import GetStaff
+
+bp_staff = Blueprint("bp_staff", __name__)
+api = Api(bp_staff)
+
+api.add_resource(GetStaff, '/staff', '/staff/<string:id>')
```

Comment:
помилки в методах PUT і DELETE через це
```suggestion
api.add_resource(GetStaff, '/staff', '/staff/<string:some_id>')
```